app/posts/2020-01-02-generate-a-page-of-screenshots.md

```diff
@@ -16,45 +16,47 @@ The first allows you to capture these screenshots from a sequence of URLs, perha
 
 1. Open `scripts/screenshot.js`. In this file you will see 2 values:
 
-    * `domain`: the website you want to screenshot, eg localhost:3000
-    * `paths`: an array of named paths
+   - `domain`: the website you want to screenshot, eg localhost:3000
+   - `paths`: an array of named paths
 
-    Replace these with your desired values.
+   Replace these with your desired values.
 
 2. In the terminal, type:
 
-    `node scripts/screenshot.js [name-of-your-feature]`
+   ```shell
+   node scripts/screenshot.js [name-of-your-feature]
+   ```
 
-    This will:
+   This will:
 
-    * visit each page lists in paths and save a screenshot
-    * save those screenshots in the named directory
-    * generate an index page with screenshots listed in order
+   - visit each page lists in paths and save a screenshot
+   - save those screenshots in the named directory
+   - generate an index page with screenshots listed in order
 
 ### Example
 
 With the following values in `scripts/screenshots.js`:
 
-``` js
-const domain = 'http://localhost:3000'
+```js
+const domain = "http://localhost:3000";
```

Comment:
Using JS examples that do not follow standardjs?

Reply:
Looks like a linter correcting something it shouldn't

Reply:
Damn it Prettier!!!

Reply:
Fixed.